tests/ballerina-test/pom.xml

```diff
@@ -84,6 +84,10 @@
             <groupId>org.testng</groupId>
             <artifactId>testng</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
```

Comment:
Don't we have to say it is a test phase dependency ?

Reply:
added the test scope